pages/players/[playerName].js

```diff
@@ -1,20 +1,68 @@
 import { useRouter } from 'next/router';
 import Navbar from '../../components/Navbar'
 import Footer from '../../components/Footer'
+import React from "react";
 
 
 
 const PlayerDetails = () => {
     
     const router = useRouter();
     const {playerName} = router.query;
-    
+    let player = "";
+    for(let ch in playerName) {
+        if (playerName[ch] != '-')
+            player += playerName[ch];
+        else
+            player += ' ';
+    }
+    player = player.toUpperCase();
+
+
+
     return (
         <>
             <Navbar/>
-                <div className='mt-10 w-full flex justify-center items-center'>
-                    <h1 className="text-3xl font-bold underline">Player page For {playerName}</h1> 
+            <div>
+                <div className="flex mt-5">
+                    <img className="flex h-200 mx-20 bg-gray-400" src="https://a.espncdn.com/combiner/i?img=/i/headshots/college-football/players/full/4429084.png" width="350" height="400"/>
```

Comment:
height should be responsive ex: fractional or percentage

Reply:
percentage of what?

Reply:
I did that so that it wasnt pressed up against the top of the screen